addons/website_sale_slides/models/product_product.py

```diff
@@ -15,4 +15,5 @@ def get_product_multiline_description_sale(self):
         if not payment_channels:
             return super(Product, self).get_product_multiline_description_sale()
 
-        return _('Access to:\n%s', '\n'.join(payment_channels.mapped('name')))
+        new_line = '' if len(payment_channels) == 1 else '\n'
+        return _('Access to: %s%s', new_line, '\n'.join(payment_channels.mapped('name')))
```

Comment:
In case we have more than 1, won't we have two `\n` before the first payment_channels name? Is this what we want?

Reply:
Joining an array of one element is the same as taking the element itself so no only one `\n`
```python
>>> 'coucou'.join(['coucou'])
'coucou'
```